### ISSUE TITLE
No permission error while using Firebase Database

### ISSUE THREAD

**Issue by linuxing3** [fullstackreact/react-native-firestack#9]
When I using the `firestack.database()` to get `key` object from `firebase real time database`,  it shows error `Client doesn't have permission to access Desired data`.
When I changed the rules in `firebase console` to no auth for reading, it works fine.
Does the database object need extra token to access database?

Rules on `firebase console`
```
{
  "rules": {
    ".read": "auth == null",
    ".write": "auth == null"
  }
}
```

Testing code
```
export const db = async(key) => {
    let snap = await firestack.database.ref(key).once('value');
    let str = JSON.stringify(snap.val());
    if (snap) {
        Alert.alert(snap.key);
        return snap.val();
    };
};
```

**Comment by auser** [comment]
This has been updated with the latest source. Can you try it using the branch `feature/modules` branch?

**Comment by auser** [comment]
I'll be pushing an update to the npm module later this week (hopefully tomorrow/Friday) if not

**Comment by auser** [comment]
Should be closed with latest push `v2.1.0`
